packages/widget/src/components/address-input/address-input.ts

```diff
@@ -5,7 +5,8 @@ import { ifDefined } from 'lit/directives/if-defined.js';
 import { Network } from '@buildwithsygma/sygma-sdk-core';
 import { when } from 'lit/directives/when.js';
 import { validateAddress } from '../../utils';
-import { BaseComponent } from '../base-component/base-component';
+
+import { BaseComponent } from '../common/base-component/base-component';
```

Comment:
can we use named exports from some `index.ts` file? In doing so we avoid having `../common/base-component/base-component` and hence we would have `../common/base-component` as the url for the import? Wdty?

Reply:
It's a good idea and I tried to do so. But somehow re-export for abstract classes doesn't work. Looks like this issue has existed for quite a long time